src/components/ImagePicker/ImagePicker.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { FC, useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { useTranslation } from 'react-i18next';
+import { Alert, Input, Button } from '@keen.io/ui-core';
+
+import {
+  InsertImage,
+  Buttons,
+  Description,
+  InputWrapper,
+  CancelButton,
+} from './ImagePicker.styles';
+
+import { saveImage } from '../../modules/queries';
+
+import { hideImagePicker } from '../../modules/app';
+
+const ImagePicker: FC<{}> = () => {
+  const dispatch = useDispatch();
+  const { t } = useTranslation();
+
+  const [error, setError] = useState(null);
+  const [link, setLink] = useState('');
+
+  const isLink = (url: string) => {
+    return /(http(s?):)([/|.|\w|\s|-])*\.(?:jpg|gif|png)/.test(url);
```

Comment:
are we considering `jpeg`, `svg` files?

Reply:
I believe this set of supported formats is enough for initial version. In my opinion this function should be move from `render` method as it's a pure function and do not need to be re-defined for each component re-render.